js assignments/calculator.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<head>
+	<script>
+		
+		function add(val){
+            var a,b,c;

```

Comment:
prefer `const` over `let` and `let` over `var`, if we are not modifying variable value further declare it as `const` else `let`